package.json

```diff
@@ -24,5 +24,11 @@
 		"@wmde/eslint-config-wikimedia-typescript": "^0.1.1",
 		"eslint-config-wikimedia": "^0.15.3",
 		"typescript": "^3.9.3"
-	}
+	},
+	"keywords": [
+		"wikibase",
+		"wikidata",
+		"datamodel",
```

Comment:
Is the "datamodel" keyword useful? I can’t really envision a scenario where someone would search for this, but not "wikibase" or "wikidata", and then be happy with finding our @wmde/wikibase-datamodel-types.

Reply:
I don't know - was under the assumption that it would certainly not hurt. If I interpret what you are saying correctly then basically all words which are already contained in the package name can/should be avoided.

Reply:
No, that’s not what I meant… the difference is that I could see someone searching for “keywords:wikibase” or “keywords:wikidata” on npmjs, and they would be interested in this library (which would come up as a result); on the other hand, I can’t imagine a case where someone would search for “keywords:datamodel” and then be glad to have found this library.

That said, I suppose it doesn’t hurt very much, so if you want to keep it, that’s also okay.

Reply:
> No, that’s not what I meant… the difference is that I could see someone searching for “keywords:wikibase” or “keywords:wikidata” on npmjs, and they would be interested in this library (which would come up as a result); on the other hand, I can’t imagine a case where someone would search for “keywords:datamodel” and then be glad to have found this library.
> 
> That said, I suppose it doesn’t hurt very much, so if you want to keep it, that’s also okay.

I can imagine searching for "wikibase datamodel" and seeing only the very relevant results, opposed to searching for "wikibase".
Anyway, I don't see any keywords listed here being a problem, only a plus.

Reply:
Removed "datamodel".
Maybe not quite in line with the commit message but I also thought of the bubbles shown on the search result page - but the buzz word is in the package name.

cf.
![Bildschirmfoto am 2020-06-09 um 18 10 20](https://user-images.githubusercontent.com/167557/84172741-a50d4b80-aa7c-11ea-99a5-4b20f9445d67.png)